actions/actions.py

```diff
@@ -302,16 +303,17 @@ async def run(
         }
 
         if tracker.get_slot("zz_confirm_form") == "yes":
-            search_type = tracker.get_slot("search_type") == "deposit"
-            vendor_name = tracker.get_slot("vendor_name")
+            search_type = tracker.get_slot("search_type")
+            vendor = tracker.get_slot("vendor_name")
+            vendor_name = f" at {vendor.title()}" if vendor else ""
             start_time = parser.isoparse(tracker.get_slot("start_time"))
             end_time = parser.isoparse(tracker.get_slot("end_time"))
             transactions = profile_db.search_transactions(
                 tracker.sender_id,
                 start_time=start_time,
                 end_time=end_time,
-                deposit=search_type,
-                vendor=vendor_name,
+                deposit=search_type=="deposit",
```

Comment:
this is a bool?

Reply:
It is. Should it rather not be?

Reply:
I think it's fine, i just found this confusing to look at at first

Reply:
maybe it'd be easier to parse if we did it above where we set all the other vars, as `deposit = True if search_type == "deposit" else False`

Reply:
ja defs